File: js/padArraySpec.js
// Write unit tests!
var char = require("./padArray");

console.log(char.padArray([1,2,3], 5, 'apple'))
console.log(char.padArray([1,2,3], 9, 'tree'))
console.log(char.padArray([1,2,3], 7, ))
console.log(char.padArray([1,2], 5, 7))
console.log(char.padArray([1], 1, 'apple'))

File: python/pad_array_spec.py
# Write unit tests!
from pad_array import pad

print(pad([1,2,3], 6)== [1, 2, 3, None, None, None])
print(pad([1,2,3], 5, "it's working")== [1, 2, 3, "it's working", "it's working"])
print(pad([1,2,3], 0,'Nope')== [1,2,3])
print(pad([1,2,3], 3,'Nope')== [1,2,3])